pkg/cmd/opts/install.go

```diff
@@ -780,7 +780,35 @@ func (o *CommonOptions) InstallGcloud() error {
 	if runtime.GOOS != "darwin" || o.NoBrew {
 		return errors.New("please install missing gcloud sdk - see https://cloud.google.com/sdk/downloads#interactive")
 	}
-	return o.RunCommand("brew", "cask", "install", "google-cloud-sdk")
+	versionResolver, err := o.GetVersionResolver()
```

Comment:
What's about adding an integration test which shows that this code works. Something along these lines:

https://github.com/jenkins-x/jx/pull/6494/files#diff-a6e8c21bcfd8a25876b6a9d4ee96e191

Reply:
Sounds cool :+1: I will give it a shot.